Service/src/routes/league.py

```diff
@@ -14,10 +14,10 @@
 @league_api.route('/api/league', methods=['POST'])
 def post_league():
    body = request.get_json()
-   try:
-      result = LeagueSchema().load(body)
-   except ValidationError as err:
-      return make_response(json.dumps(err.messages), 400)
+   # try:
```

Comment:
again, no commented code, remove it if it's unused

beauty of Git/Github is that history is preserved

Reply:
Removed